src/sentry/integrations/jira/client.py

```diff
@@ -215,3 +217,8 @@ def transition_issue(self, issue_key, transition_id):
     def assign_issue(self, key, name_or_account_id):
         user_id_field = self.user_id_field()
         return self.put(self.ASSIGN_URL % key, data={user_id_field: name_or_account_id})
+
+    def get_email(self, name_or_account_id):
+        user_id_field = self.user_id_field()
```

Comment:
Should this be hardcoded to `accountId` as that is the only accepted parameter that Atlassian documents?